docs/users.md

```diff
@@ -29,6 +30,15 @@ client.users.update('123', {name: 'New Name', job_title: 'New Title', phone: '55
 ```
 
 
+Invite User
+-----------
+
+To invite an existing user to join an Enterprise call the [`users.invite(enterpriseID, login, callback)`](http://opensource.box.com/box-node-sdk/Users.html#invite) method.
```

Comment:
It might be worth adding here that this call requires additional scopes for the calling app, e.g.:

> This method requires the "Manage An Enterprise" scope for the enterprise, which can be enabled within your developer console."